pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -148,10 +148,6 @@ public String getLabelTooltip() {
   }
 
   protected void closeTab() {
-    if ( getContent().getElement().getElementsByTagName( "iframe" ).getLength() > 0 ) {
```

Comment:
Why do you remove that now? We still want to know when a tab is closed to issue the cancel request.